canonical-web-and-design/ubuntu.com
# Update /what-is-kubeflow as per copydoc

## Done

- Update /what-is-kubeflow as per [copydoc](https://docs.google.com/document/d/1m-96RlGkbzFa1KaGAxYFpY0ztIlsTuViCjXSAlErQlg/edit#)

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/ai/what-is-kubeflow
    - Be sure to test on mobile, tablet and desktop screen sizes
- Compare against [copydoc](https://docs.google.com/document/d/1m-96RlGkbzFa1KaGAxYFpY0ztIlsTuViCjXSAlErQlg/edit#)

## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/4905
